uiowa/uiowa
# 91st Meridian - Switched author bio field to use filtered HTML and other cleanup

Resolves #6079 
Also cleans up the `core.entity_view_display.node.article.default` config for both the periodical feature and the 91st Meridian site split. I re-enabled the display of the article creation date in the periodical feature because I was not able to figure out how to selectively not display it.

# How to test
```
ddev blt ds && ddev drush @default.local config-split:activate periodical -y && ddev drush @default.local cim -y && ddev drush @default.local uli node/add/article
```
- As a periodical content editor, I can insert an image on the author biography field in the article content type.
- Validate that the created on date shows
```
ddev blt ds --site 91stmeridian.iwp.uiowa.edu && ddev drush @iwp91stmeridian.local uli issues/volume-6-issue-1-61-spring-2008/last-bird-burial-master
```
- Validate that the created on date shows
- Check that nothing has changed from https://91stmeridian.iwp.uiowa.edu/issues/volume-6-issue-1-61-spring-2008/last-bird-burial-master
